server/functions/framework/compiled_function.go

```diff
@@ -59,10 +60,20 @@ func (c *CompiledFunction) String() string {
 	sb := strings.Builder{}
 	sb.WriteString(c.Name + "(")
 	for i, param := range c.Parameters {
+		// Aliases will output the string "x as x", which is an artifact of how we build the AST, so we'll bypass it
+		if alias, ok := param.(*expression.Alias); ok {
+			param = alias.Child
+		}
 		if i > 0 {
 			sb.WriteString(", ")
 		}
-		sb.WriteString(param.String())
+		if doltgresType, ok := param.Type().(pgtypes.DoltgresType); ok && pgtypes.ShouldQuoteType(doltgresType.BaseID()) {
```

Comment:
Probably just replace with `pgtypes.QuoteTypeName` that decides whether quoting is necessary or not